packages/client/src/comp/states/NewHampshire.tsx

```diff
@@ -1,22 +1,16 @@
 import React from 'react'
 
-import Select from 'muicss/lib/react/select'
-import Option from 'muicss/lib/react/option'
-
-import { NewHampshireInfo, isNewHampshirePrimaryParty, newHampshirePrimaryParty, primaryEligible } from '../../common'
+import { Select } from '../util/Select'
+import { NewHampshireInfo, newHampshirePrimaryParty, primaryEligible, NewHampshirePrimaryParty } from '../../common'
 import { SignatureBase } from './Base'
-import { useControlRef } from '../util/ControlRef'
 
 export const NewHampshire = () => {
-  const primaryPartyRef = useControlRef<Select>()
+  const [ primaryParty, setPrimaryParty ] = React.useState<NewHampshirePrimaryParty>(newHampshirePrimaryParty[0])
```

Comment:
Use the actual value instead of `newHampshirePrimaryParty[0]` -- it's more explicit and still checked by the type system.